cinema/views.py

```diff
@@ -1,6 +1,9 @@
+from datetime import datetime
```

Comment:
Accorfing to pep8 blank line should be between different import groups 

`from datetime import datetime `
And 
`from django.db.models import Count, F`
are different import groups